tests/generation/test_generation_utils.py

```diff
@@ -1765,22 +1782,21 @@ def test_contrastive_search_t5(self):
         self.assertListEqual(
             generated_text,
             [
-                """Liana Barrientos has been married 10 times, nine of them in the Bronx. Her husbands filed for permanent residence after the marriages, prosecutors say."""
+                'Liana Barrientos has been married 10 times, nine of them in the Bronx. Her husbands filed for '
+                'permanent residence after the marriages, prosecutors say.'
             ],
         )
 
     @slow
     def test_contrastive_search_opt(self):
-        article = r"""A chat between a curious human and the Statue of Liberty.
-
-Human: What is your name?
-Statue: I am the Statue of Liberty.
-Human: Where do you live?
-Statue: New York City.
-Human: How long have you lived there?"""
+        article = (
```

Comment:
on this test:
- Formatted to comply with the 120 char limit
- changed the model to a smaller one, to fit in CI